src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -326,12 +323,7 @@ public void setField(String name, String value) {
         Objects.requireNonNull(name, "field name must not be null");
         Objects.requireNonNull(value, "field value must not be null");
 
-        if (value.isEmpty()) {
```

Comment:
Why is this code removed? See https://github.com/JabRef/jabref/pull/1281#discussion_r60948627 for a discussion about the addition of this code.